### PR TITLE
Xeno starting number and latejoin rebalance, monkey removal, survivor rework, admin xeno rolling fix.

### PR DESCRIPTION
**Pull request by DominikPanic**
:cl: LaKiller8
balance: Slightly increased the amount of starting xenos, increased bonuses for xenos when marines late-join.
balance: Monkeys are now completely removed.
balance: Survivor loadouts are now more consistent, they start with more useful materials and ammo for their basic gun.
fix: Fixed rolling of xenos for admins.
/:cl:

### CODE REVIEW

**Comment by Rohesie** [comment]
Multiplying is better than dividing in general.
This will have to be tested for balance.

**Comment by DominikPanic** [comment]
> 
> 
> Multiplying is better than dividing in general.
> This will have to be tested for balance.

I'd say that in this case it's easier to comprehend since we're dividing by pretty high numbers. (not here specifically but in the whole file) making it easier to compare each change.

File: code/__HELPERS/game.dm [hunk 1]
```diff
@@ -293,27 +293,24 @@ proc/isInSight(var/atom/A, var/atom/B)
...
 
-	for (var/mob/dead/observer/O in player_list)
-		// Jobban check
-		if (!O.client || !O.client.prefs || !(O.client.prefs.be_special & BE_ALIEN) || jobban_isbanned(O, "Alien"))
-			continue
-
-		// copied from join as xeno
-		var/deathtime = world.time - O.timeofdeath
-		if(deathtime < 3000 && ( !O.client.holder || !(O.client.holder.rights & R_ADMIN)) )
+	for(var/mob/dead/observer/O in player_list)
```
Comment (spookydonut) [inline code/__HELPERS/game.dm]:
```suggestion
	for(var/mob/dead/observer/O in dead_mob_list)
```

File: code/__HELPERS/game.dm [hunk 2]
```diff
@@ -293,27 +293,24 @@ proc/isInSight(var/atom/A, var/atom/B)
...
-				if("No")
-					continue
+		//Admins get to skip the deathtime check
+		if(O.client?.holder?.rights && (O.client.holder.rights & R_ADMIN))
+			candidates += O.key
+			continue
+
+		//Recently dead non-admins cannot be drafted.
+		var/deathtime = world.time - O.timeofdeath
+		if(deathtime < 3000)
```
Comment (spookydonut) [inline code/__HELPERS/game.dm]: magic number, use MINUTES

File: code/game/gamemodes/cm_initialize.dm [hunk 3]
```diff
@@ -90,9 +90,9 @@ Additional game mode variables.
 
 datum/game_mode/proc/initialize_special_clamps()
 	var/ready_players = ready_players() // Get all players that have "Ready" selected
-	xeno_starting_num = max((ready_players/7), xeno_required_num)
-	surv_starting_num = CLAMP((ready_players/25), 0, 8)
-	merc_starting_num = max((ready_players/3), 1)
+	xeno_starting_num = max((ready_players / 6), xeno_required_num)
```
Comment (spookydonut) [inline code/game/gamemodes/cm_initialize.dm]: Move these magic numbers out for defines

File: code/game/gamemodes/cm_initialize.dm [hunk 4]
```diff
@@ -68,7 +68,7 @@ Additional game mode variables.
 	var/monkey_amount		= 0 //How many monkeys do we spawn on this map ?
 	var/list/monkey_types	= list() //What type of monkeys do we spawn
 	var/latejoin_tally		= 0 //How many people latejoined Marines
-	var/latejoin_larva_drop = 7 //A larva will spawn in once the tally reaches this level. If set to 0, no latejoin larva drop
+	var/latejoin_larva_drop = 5 //A larva will spawn in once the tally reaches this level. If set to 0, no latejoin larva drop
```
Comment (spookydonut) [inline code/game/gamemodes/cm_initialize.dm]: replace this variable with a static define.

File: code/game/gamemodes/cm_initialize.dm [hunk 5]
```diff
@@ -68,7 +72,7 @@ Additional game mode variables.
 	var/monkey_amount		= 0 //How many monkeys do we spawn on this map ?
 	var/list/monkey_types	= list() //What type of monkeys do we spawn
 	var/latejoin_tally		= 0 //How many people latejoined Marines
-	var/latejoin_larva_drop = 7 //A larva will spawn in once the tally reaches this level. If set to 0, no latejoin larva drop
+	var/latejoin_larva_drop = LATEJOIN_LARVA_REQUIRED_NUM
```
Comment (spookydonut) [inline code/game/gamemodes/cm_initialize.dm]: sorry to be clear remove the variable latejoin_larva_drop and replace all references with that define

Comment (DominikPanic) [reply code/game/gamemodes/cm_initialize.dm]: Had to do it like this.

**Comment by Rohesie** [comment]
Waiting on @spookydonut to finish the review.

**Comment by Rohesie** [comment]
This is tested and approved. Will wait on @spookydonut 's ok to merge.

**Pull request merged**